docs/modules/cluster-performance/pages/performance-tuning.adoc

```diff
@@ -1,8 +1,8 @@
 = Performance Tuning
 
-To achieve a good performance in your Hazelcast deployment, it is crucial to tune your
-production environment. This section provides guidelines for tuning the performance.
-Of course, we also recommend to run performance and stress tests to evaluate the application performance.
+To achieve good performance in your Hazelcast deployment, it is crucial to tune your
```

Comment:
```suggestion
You must tune your production environment to achieve the best performance in your Hazelcast deployment.
```

Reply:
Had a minor fix to this in main already so resolving...